spec/controllers/skills_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe SkillsController do
+  describe 'Update Skill' do
+    render_views
+
+    it 'should switch category' do
+      edited_skill = {**Skill.first.as_json.symbolize_keys,
+                      category_parent: Category.where(title: "System-Engineering")[0].id}
```

Comment:
acccess by fixtures ! skills(:system_engineering)

see fixture access helpers in rails doc

or just construct the hash here manually ... transforming given entry to params makes it less readable ...